### PR TITLE
Correction in abbreviation

### PR DESCRIPTION
**Pull request by amalkurup89**
**What this PR does / why we need it**: Documentation fix

**Which issue(s) this PR fixes**: Correction in abbreviation

### CODE REVIEW

**Comment by amalkurup89** [comment]
@oddlittlebird Can you please review

**Comment by amalkurup89** [comment]
@oddlittlebird gentle reminder for approval 🙂

File: docs/sources/guides/whats-new-in-v5.md [hunk 1]
```diff
@@ -129,7 +129,7 @@ Read more on [Graphite Tag Support](http://graphite.readthedocs.io/en/latest/tag
 
 <div class="clearfix"></div>
 
-## Dashboard model, persistent url's and API changes
+## Dashboard model, persistent URL's and API changes
```
Comment (oddlittlebird) [inline docs/sources/guides/whats-new-in-v5.md:132]:
```suggestion
## Dashboard model, persistent URLs and API changes
```

**Review by oddlittlebird — CHANGES_REQUESTED** [review]
Great catches! One minor change, then this is good to go.

**Comment by amalkurup89** [comment]
> Great catches! One minor change, then this is good to go.

@oddlittlebird  Thanks for the suggestion. I have committed it, as well

**Review by oddlittlebird — APPROVED** [review]
Rock and roll! Thanks for contributing to the docs!

**Pull request merged**